core/capabilities/remote/target/client_test.go

```diff
@@ -22,8 +22,7 @@ import (
 )
 
 func Test_Client_DonTopologies(t *testing.T) {
-	ctx, cancel := context.WithCancel(testutils.Context(t))
-	defer cancel()
+	ctx := testutils.Context(t)
```

Comment:
This helper already registers `tb.Cleanup(cancel)`, so we don't need to defer an explicit cancellation.